app/cafe.py

```diff
@@ -0,0 +1,25 @@
+import datetime
+from app.errors import (NotVaccinatedError,
+                        OutdatedVaccineError,
+                        NotWearingMaskError)
```

Comment:
Place each import on a new line so that there would be no imports on the same line with opening and closing parenthesis. It will help to improve the code style and readability